src/components/messages/message.tsx

```diff
@@ -32,11 +32,30 @@ export const Message: FC<Props> = ({ message, color, characterName }) => {
           <Typography variant="body1" textAlign={textAlign}>
             {message.message}
           </Typography>
-          <Typography variant="caption" textAlign={"right"}>
-            {timeStamp}
-          </Typography>
+          <Stack>
+            <Typography variant="caption" textAlign={"right"}>
+              {timeStamp}
+            </Typography>
+          </Stack>
         </Stack>
       </Stack>
     </Stack>
   );
 };
+
+const getCharacter = (characterName: string) => {
```

Comment:
Maybe we need a global util for getting the characters, since we're using it in several places now

Reply:
Done